app/invocation/invocation_action_card.tsx

```diff
@@ -74,7 +74,11 @@ export default class InvocationActionCardComponent extends React.Component<Props
   }
 
   fetchActionResult() {
-    const digest = parseDigest(this.props.search.get("actionDigest"));
+	let digestParam = this.props.search.get("actionResultDigest");
+	if (digestParam == null) {
+	    digestParam = this.props.search.get("actionDigest");
+	}
```

Comment:
Looks like this needs formatting; can you run `./buildfix.sh` to fix it?

Reply:
Done.